faer/src/mat/matref.rs

```diff
@@ -1377,13 +1377,9 @@ where
 	/// # Examples
 	///
 	/// ```
-	/// use faer::{mat, Mat};
+	/// use faer::{Mat, mat};
 	///
-	/// let m = mat![
-	///     [1.0, 5.0, 3.0],
-	///     [4.0, 2.0, 9.0],
-	///     [7.0, 8.0, 6.0],
-	/// ];
+	/// let m = mat![[1.0, 5.0, 3.0], [4.0, 2.0, 9.0], [7.0, 8.0, 6.0],];
```

Comment:
Looks like this got formatted, I assume we don't want this? Have you tried doing a `#[rustfmt::skip]` on this?

Regardless, I can discard these changes.

Reply:
294f3b6b74a5c511e33d0eb60e95e26ae7a9562c

Reply:
probably not a big deal